pkg/console/util_test.go

```diff
@@ -79,17 +79,11 @@ func TestGetFormattedServerURL(t *testing.T) {
 			output: "https://example.org:443",
 			err:    nil,
 		},
-		{
-			Name:   "invalid ip",
-			input:  "1.2.3.4/",
```

Comment:
Does RKE2 support this format？

Reply:
It's a silly mistake for not checking path in url, added now.